viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -15,8 +15,7 @@ import {ReportRenderer} from '../../../report/renderer/report-renderer.js';
 import {TextEncoding} from '../../../report/renderer/text-encoding.js';
 import {renderFlowReport} from '../../../flow-report/api';
 
-// @ts-expect-error Legacy use of report renderer
-const dom = new DOM(document);
+const dom = new DOM(document, document.documentElement);
```

Comment:
Could we call this `documentDom` or `rootDom` or something to differentiate from the local instance created below